packages/test_track/example/main.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:test_track/test_track.dart';
+
+Future<void> main() async {
```

Comment:
I decided to just do a simple dart example instead of a Flutter one. We can always come back through and add a separate main.dart and a pubspec.yaml to show a flutter example. It will mostly be the same as this but with a widget tree 😄